front/src/modules/timesStops/TimesStopsOutput.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+
+import cx from 'classnames';
+
+import type {
+  PathPropertiesFormatted,
+  SimulationResponseSuccess,
+} from 'applications/operationalStudies/types';
+import type { TrainScheduleResult } from 'common/api/generatedEditoastApi';
+import type { OperationalPointWithTimeAndSpeed } from 'modules/trainschedule/components/DriverTrainScheduleV2/types';
+import type { PathStep } from 'reducers/osrdconf/types';
+import { NO_BREAK_SPACE } from 'utils/strings';
+
+import useOutputTableData from './hooks/useOutputTableData';
+import TimesStops from './TimesStops';
+import { TableType, type PathWaypointRow } from './types';
+
+type TimesStopsOutputProps = {
+  simulatedTrain: SimulationResponseSuccess;
+  pathProperties: PathPropertiesFormatted;
+  operationalPoints: OperationalPointWithTimeAndSpeed[];
+  selectedTrainSchedule: TrainScheduleResult;
+  startTime: string;
+  pathSteps: PathStep[];
+  pathLength?: number;
```

Comment:
I think pathLength should be not nullable

Reply:
it comes from useSimulationResults